src/storage/v2/durability/wal.hpp

```diff
@@ -157,6 +160,8 @@ struct WalDeltaData {
     std::string label;
   } operation_text;
 
+  VectorIndexSpec operation_vector;
```

Comment:
Also the massive object problem (not a union), what about this? @Ignition

Reply:
@gitbuda I'm look at refactoring WalDeltaData today

Reply:
@DavIvek @gitbuda refactor is #2517